fava/static/javascript/transaction-overlay.js

```diff
@@ -42,6 +42,12 @@ export default function showTransactionOverlay() {
   if (!initialized) {
     addPostingRow(form);
     addPostingRow(form);
+    $.delegate(form, 'keyup', '.number', (event) => {
```

Comment:
This should go in `entry-forms.js` (otherwise the ingest and overlay form behave differently)

Reply:
I moved this to the "global" level in https://github.com/beancount/fava/pull/623/commits/eeb60a6ef9d4d5c3cfefb865e3bbbf894d4b0574, because the Ingest entry forms are not initialized with JavaScript, but the overlay one is.